src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -603,10 +604,15 @@ export class SettingsEditor2 extends EditorPane {
 
 		const actionBar = this._register(new ActionBar(this.controlsElement, {
 			animated: false,
-			actionViewItemProvider: (_action) => { return undefined; }
+			actionViewItemProvider: (_action) => {
```

Comment:
I usually just use the `_` prefix for arguments that aren't used, so lint doesn't complain, so you could change it to `action` or something now